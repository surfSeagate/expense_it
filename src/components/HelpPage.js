import React from 'react';

const HelpPage = () => (
  <div>
    This is my add help page!
  </div>
);

export default HelpPage;