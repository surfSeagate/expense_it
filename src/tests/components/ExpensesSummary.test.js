import React from 'react';
import { ExpensesSummary } from '../../components/ExpensesSummary';
import numeral from 'numeral';
import { shallow } from 'enzyme';

test('should correctly render exepnses with 1 expense', () => {
  const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={235} />);
  expect(wrapper).toMatchSnapshot();

});
test('should correctly render exepnses with multiple expenses', () => {
  const wrapper = shallow(<ExpensesSummary expenseCount={23} expensesTotal={23534134134} />);
  expect(wrapper).toMatchSnapshot();

});