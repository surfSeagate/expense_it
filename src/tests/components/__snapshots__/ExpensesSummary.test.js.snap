// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render exepnses with 1 expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totaling 
    $2.35
  </h1>
</div>
`;

exports[`should correctly render exepnses with multiple expenses 1`] = `
<div>
  <h1>
    Viewing 
    23
     
    expenses
     totaling 
    $235,341,341.34
  </h1>
</div>
`;
